smartdevicelink/sdl_java_suite
# Fix an issue with starting video stream after closing the app

Fixes #1248 

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan
* Start video stream on a SYNC 3 head unit
* Close the app using voice commands
* Open the app again and start the video stream
* Make sure streaming works correctly 

### Summary
This PR handles a bug in some head units where the `StartService` for video streaming gets a `NAK` response after the app is being closed using voice commands. 
The PR fixes that by manually sending `EndService` from app side when the app is closed using voice commands. 

### CLA
